test/UnitTests/CheckoutControllerTest.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Security.Claims;
+using System.Threading;
+using Microsoft.AspNet.Http.Core;
+using Microsoft.AspNet.Http.Core.Collections;
+using Microsoft.AspNet.Mvc;
+using Microsoft.AspNet.Routing;
+using Microsoft.Framework.DependencyInjection;
+using Microsoft.Framework.DependencyInjection.Fallback;
+using Xunit;
+using MusicStore.Controllers;
+using MusicStore.Models;
+
+namespace UnitTests
```

Comment:
@ianhong MusicStore tests were written long back and that time I did not know about these guidelines. I have been fixing some of these while I write new code. So feel free to fix the unit tests project to follow guidelines in newer projects.